src/azul/chalice.py

```diff
@@ -189,15 +189,30 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
+    hsts_max_age = 60 * 60 * 24 * 365 * 2
+
+    response_headers = {
```

Comment:
These are all security-related headers so the name of the variable should reflect that.